### ISSUE TITLE
Feature Request - Provide More Safeguards for Library Deletion

### ISSUE THREAD

**Issue by Nitrousoxide** [gotson/komga#1399]
### Describe your suggested feature

Library Delete is a HIGHLY destructive action, and it is currently only a couple of options away from a frequently used "empty trash" option.  It is extremely easy for a user who isn't paying close attention to what they are doing to mistake the two, or to mindlessly attempt to use that option to delete some selected series in the manga list (I certainly did!  But luckily I keep good backups!)

This option obviously has a popup to confirm, but it's a similar one to the "empty trash" popup confirmation and extremely easy to mistake it for the other.

Given the fact that this completely blows out all data for that library I really feel like additional protections need to be in place.  Even something as simple as requiring the user to type out the name of the library and making the confirm button bright red would serve to distinguish it from "empty trash" and from the "delete series" which are also both accessible from the series list.

Ideally I think it would be best to move this option entirely out of the normal workflow's view and relegate it to somewhere in the server options since it's not the sort of setting you should be using in day-to-day use.

<img width="452" alt="image" src="https://github.com/gotson/komga/assets/1363931/8192986f-50c0-4139-b6f8-3363345cb071">
<img width="450" alt="image" src="https://github.com/gotson/komga/assets/1363931/0335e7ee-2c11-4539-9efd-653cc07adeee">
<img width="443" alt="image" src="https://github.com/gotson/komga/assets/1363931/f66e9712-917e-45e6-90ab-00392bf7732c">

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I have updated the app to the latest version.
- [X] I will fill out all of the requested information in this form.

**Comment by gotson** [comment]
"users don't read".

**Comment by AnnoyedDev** [comment]
A good Idea would be to do the same thing as github do for example wich mean we have to WRITE the full name of the Library name, with case sensitivy and paste disabled. I think it could be a very good idea for user that "don't read" :/

**Comment by gotson** [comment]
yeah or users can start reading. if they wipe their library because they don't read big warnings in red and they tick confirmation boxes without reading, that will probably generate enough harm so that they realize they should read stuff :D

**Comment by Nitrousoxide** [comment]
That seems like an... unproductive... approach to UI design.
"The users will hurt themselves until they share our vision"

**Comment by gotson** [comment]
> That seems like an... unproductive... approach to UI design. "The users will hurt themselves until they share our vision"

if it gets implemented, there's still a good chance people will still complain that it's not enough, that's all i'm saying. I haven't closed that issue, but there's a lot of other more important stuff to do.

**Comment by billsargent** [comment]
> That seems like an... unproductive... approach to UI design. "The users will hurt themselves until they share our vision"

For this very reason, I keep a backup of my stuff. I mean I use this from a tablet... one slip up and yeah. Gone.
